flexget/components/managed_lists/lists/ombi_list.py

```diff
@@ -152,16 +152,16 @@ def __init__(
             self.ombi_title = self.ombi_title + ' E' + str(data['tmdb_episode']).zfill(2)
 
     @property
-    def requestId(self) -> str:
+    def request_id(self) -> str:
```

Comment:
I'm a bit torn on this. Sometimes I think it's okay to use camelcase when we are mirroring another api.

Reply:
Since this rule applies in most cases, `noqa` can be used for the rare exceptions.

Reply:
I think actually this particular instance is fine to just be lower case.